src/Eori/EoriValidator.php

```diff
@@ -2,21 +2,28 @@
 
 namespace CrazyFactory\Validation\Eori;
 
-use CrazyFactory\HarmonizedSystem\Countries;
+use SoapClient;
 
 class EoriValidator
 {
+    private $soapClient;
+
+    public function __construct(SoapClient $client = null)
+    {
+        $this->soapClient = $client ?? new SoapClient('https://ec.europa.eu/taxation_customs/dds2/eos/validation/services/validation?wsdl');
```

Comment:
add a connection check to separate 
- Exception from input data
  - We will throw a specific exception and handle it on the implementation.
- Exception from timeout/service not available
  - We will mark it as success and catch it for logging on the implementation

Reply:
Oh just have a closer look at the exception.
There is a faultcode. Maybe we can use this. But this need to be caught in userland
![image](https://user-images.githubusercontent.com/26110220/59417891-43c38580-8df2-11e9-8ed5-19445a86fdaf.png)

Reply:
OK Then